### PR TITLE
[HOPSWORKS-751] Add support for sklearn model serving

### PR DESCRIPTION
**Pull request by Limmen**
- materialize sklearn scripts to hopsworks bin dir
- rename tfserving table to serving
- rename columns
- add attributes for sklearn serving

### CODE REVIEW

**Comment by Limmen** [comment]
Tests: https://github.com/logicalclocks/hops-testing/pull/385

File: templates/default/sql/dml/0.10.0.sql.erb [hunk 1]
```diff
@@ -30,3 +30,9 @@ REPLACE INTO `hopsworks`.`variables` VALUES ("kagent_liveness_threshold", "<%= n
...
 REPLACE INTO `hopsworks`.`variables` VALUES ("service_start_script", "<%= node['hopsworks']['kagent_liveness']['start_script'] %>");
 REPLACE INTO `hopsworks`.`variables` VALUES ("service_stop_script", "<%= node['hopsworks']['kagent_liveness']['stop_script'] %>");
+
+
+REPLACE INTO `hopsworks`.`variables` VALUES ("sklearn_serving_connection_pool_size", "<%= node['sklearn_serving']['pool_size'] %>");
+REPLACE INTO `hopsworks`.`variables` VALUES ("sklearn_serving_max_route_connections", "<%= node['sklearn_serving']['max_route_connections'] %>");
+
+REPLACE INTO `hopsworks`.`schema_topics`(`name`, `version`, `contents`) VALUES ('inferenceschema', 1, '{"fields": [{"name": "modelId", "type": "int"}, { "name": "modelName", "type": "string" }, {  "name": "modelVersion",  "type": "int" }, {  "name": "requestTimestamp",  "type": "long" }, {  "name": "responseHttpCode",  "type": "int" }, {  "name": "inferenceRequest",  "type": "string" }, {  "name": "inferenceResponse",  "type": "string" }, { "name": "servingType", "type": "string" } ],  "name": "inferencelog",  "type": "record" }');
```
Comment (SirOibaf) [inline templates/default/sql/dml/0.10.0.sql.erb]: We should bump the version, otherwise it will break current installations, as the schema is different.

Comment (Limmen) [reply templates/default/sql/dml/0.10.0.sql.erb]: Done

File: templates/default/sklearn_flask_server.py.erb [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+
+INPUTS = "inputs"
+global prediction_service
+
+def parse_args():
+    """Parses the commandline arguments with argparse"""
+    parser = argparse.ArgumentParser()
+    parser.add_argument("-p", "--port", help="port that the flask server will listen to",
+                        default=5000)
+    parser.add_argument("-f", "--file", help="ip that the flask server will listen to",
```
Comment (SirOibaf) [inline templates/default/sklearn_flask_server.py.erb]: Duplicated help string.

Comment (Limmen) [reply templates/default/sklearn_flask_server.py.erb]: Done

**Pull request merged**